src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -393,7 +403,7 @@ def set_timesteps(
         sigmas = np.concatenate([sigmas, [sigma_last]]).astype(np.float32)
 
         self.sigmas = torch.from_numpy(sigmas)
-        self.timesteps = torch.from_numpy(timesteps).to(device=device, dtype=torch.int64)
```

Comment:
This is one big difference -> instead of returning integers, it returns float between 0 and 1!